frontend/src/pages/GamePage.tsx

```diff
@@ -1,25 +1,26 @@
-import React, { useState } from "react";
+// import { quizQuestions } from "./questions";
+
+import React, { useEffect, useState } from "react";
 import Terminal, { ColorMode, TerminalOutput } from "react-terminal-ui";
-import { quizQuestions } from "./questions";
+import axios from "axios";
 
 const GamePage: React.FC = () => {
-    const [terminalLineData, setTerminalLineData] = useState([
+    const [terminalLineData, setTerminalLineData] = useState<React.ReactNode[]>([
         <TerminalOutput>Welcome to the Life Choices Game!</TerminalOutput>,
         <br />,
-        <TerminalOutput>Your initial stats: Health: 50, Happiness: 50, Money: 50.</TerminalOutput>,
-        <br />,
-        <TerminalOutput>{`Question 1: ${quizQuestions[0].question}`}</TerminalOutput>,
-        ...quizQuestions[0].options.map((option, index) => (
-            <TerminalOutput key={`tout-${index}`}>{`${index + 1}. ${option.text}`}</TerminalOutput>
-        )),
+        <TerminalOutput>Loading questions, please wait...</TerminalOutput>,
```

Comment:
Instead of rendering loading... text use the Loading component defined in /components/loading.tsx
Or, if you can do, try to create a new component named, terminal-loading.tsx which shows a loading component like that is displayed on terminal... Check this out: https://stackoverflow.com/questions/34848505/how-to-make-a-loading-animation-in-console-application-written-in-javascript-or